### ISSUE TITLE
Enable/disable for site

### ISSUE THREAD

**Issue by peirix** [craftcms/digital-products#66]
### Description

It would be really nice to be able to disable a product for a specific site

**Comment by AndrewMac** [comment]
Yes, please. Multi-site is a core component of Craft CMS, and I would like to see support in Digital Products. And I would also like to see support for assigning products to the relevant store in multi-store setups as well. Thanks!
